xml/System.Numerics/Vector.xml

```diff
@@ -3616,8 +3632,8 @@ Note that this method returns a <xref:System.Single> instead of an integral type
       <Docs>
         <param name="source1">The first source vector, whose elements become the lower-index elements of the return value.</param>
         <param name="source2">The second source vector, whose elements become the higher-index elements of the return value.</param>
-        <param name="low">To be added.</param>
-        <param name="high">To be added.</param>
+        <param name="low">The vector that will be narrowed to the lower half of the result vector.</param>
```

Comment:
Probably better if these have the same definitions as the old parameter names, i.e.

```suggestion
        <param name="low">The first source vector, whose elements become the lower-index elements of the return value.</param>
```

Reply:
I am not familiar with this topic. @tannergooding would you agree with these suggestions?

Reply:
It's weird that the extra summaries changed so much. Not sure if it's because of behavioral change of the underlying API or the purpose of the parameters changed. Maybe it's just me, but the two descriptions don't sound too similar, so I'm not convinced copying the old definition is the appropriate change.

Reply:
I don't have a preference here, they both effectively describe the same behavior.

The notable difference is that the new description explicitly states that a narrowing occurs.

Reply:
Sounds good. I'll edit my suggestion to use the new text but with "source" added.